### ISSUE TITLE
<11% mAP with pretrained YOLO

### ISSUE THREAD

**Issue by half-potato** [eriklindernoren/PyTorch-YOLOv3#278]
Only 200 iterations of training on COCO completely destroys the accuracy of YOLO. I am using an unchanged repo with the COCO dataset downloaded using the script and version 1.1.0 of pytorch.

Why does this happen? Do I actually need to train it from scratch completely?

**Comment by fadi212** [comment]
Yes. It has updated the weights file :D
Now just remove the yolov3.weights file and replace it with the new file. 
I think this is the only explanation from me
